.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,30 @@
+name: Continuous Integration
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '42 15 * * *'
+jobs:
+  build:
+    name: Build and Test
+    strategy:
+      matrix:
+        os: [ ubuntu ]
+        hhvm: [ '4.88' ]
```

Comment:
does this need to be "matrix" if it's just 1 element each?

Reply:
we could move it to an env section, but I like the consistency with other projects